src/litserve/specs/openai.py

```diff
@@ -105,6 +105,31 @@ class ToolCall(BaseModel):
     function: FunctionCall
 
 
+class ResponseFormatText(BaseModel):
+    type: Literal["text"]
+
+
+class ResponseFormatJSONObject(BaseModel):
+    type: Literal["json_object"]
+
+
+class JSONSchema(BaseModel):
+    name: str
+    description: Optional[str] = None
+    schema: Optional[Dict[str, object]] = None
```

Comment:
Hi @aniketmaurya ,

I'm encountering the following warning, due to a field in JSONSchema:

*Warning: Field name 'schema' in 'JSONSchema' shadows an attribute in parent 'BaseModel'.*

I would appreciate any suggestions you might have for resolving this issue. Is there a preferred way to handle such cases?

Thank you for your help!

Reply:
hi @bhimrazy, sorry for late response. somehow I missed the ping. Have you tried it with [alias](https://stackoverflow.com/questions/76728740/two-fields-with-same-name-causing-problem-in-pydantic-model-classes) in Pydantic?

Reply:
No worries @aniketmaurya and Thanks for the suggestion I hadn't considered using an alias. I'll give that a try and see if it resolves the warning. Appreciate the help! 🙂